modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/dmf/DMFExporter.java

```diff
@@ -1517,6 +1712,32 @@ public int length() {
         }
     }
 
+    private static final class TileData {
+        public final Map<String, DMFMapTile.TileTextureInfo> textures;
+        public final Point gridCoordinate;
+        public Vector3f bboxMin;
+        public Vector3f bboxMax;
```

Comment:
Instances of `Vector3f` are mutable. Consider making this class a record.

Reply:
I cannot make it a record since it is updated in more than one place.